### ISSUE TITLE
Supplement files handling

### ISSUE THREAD

**Issue by laclaro** [jlegewie/zotfile#332]
Hi

Often I add a journal article along with its supplement to a reference in Zotero. Zotfile renames them then such that one cannot tell which one is the paper and which one is the supplement.

I am aware of http://zotfile.com/index.html#renaming-rules but I cannot see how to achieve this.

Suggestion: Define strings (or a regexp) that are preserved in file-naming if they appear at the end of the filename (w/o ext).
Default: "paper, supplement, suppl, comments"

Henning

**Comment by bdcaf** [comment]
Also have this problem. In my case it's errata that I need to add to articles/books.

**Comment by nhardy01** [comment]
I would also appreciate this feature

**Comment by e-clin** [comment]
So glad to see that someone requested this feature before, but a bit disappointed that it's never attended to for >2 yrs! I have the same issue - when saving supplementary materials alongside the main article and using Zotfile to rename them, all files appear to bear the same name in Zotero, even if they have numbered suffixes in Windows File Explorer. I wish there is an option in the renaming contextual menu to specify the type of the document as suggested by @laclaro .

**Comment by element4l** [comment]
This is a real problem. It means without any programmed solution, we must do something like manually check every one of the tens of thousands of items and tag those that have more than one attached file before doing any kind of move and rename. The entire zotero workflow is still extremely convoluted, it is rather surprising how much struggle there is to get basic functionality in this software. I am really beginning to think about moving to something else at this point.

**Comment by boyoungzheng** [comment]
This is a really needed feature. If it is difficult to find a general strategy to distinguish main/supplementary/errata/solutions/etc files, a pop-up window during the renaming process for the manual input is acceptable.

**Comment by Galileji** [comment]
I would also personally use this feature

**Comment by szymon28** [comment]
It would be a great feature!

**Comment by menarest** [comment]
yes, would be great to have this feature! having the same problem
